presto-tests/src/main/java/com/facebook/presto/tests/TestedFeature.java

```diff
@@ -21,18 +21,25 @@
 
 public enum TestedFeature
 {
+    DATE_TYPE,
+
     SCAN,
     JOIN,
     ORDER_BY,
-    DELETE,
-
     FILTERED_AGGREGATION,
     AGGREGATION(FILTERED_AGGREGATION),
+    DELETE,
+    INSERT,
 
     CREATE_TABLE,
-
-    METADATA(ADD_COLUMN, DROP_COLUMN, CREATE_TABLE),
-    CONNECTOR(SCAN, METADATA),
+    DROP_TABLE,
+    RENAME_TABLE,
+    ADD_COLUMN,
+    DROP_COLUMN,
+    RENAME_COLUMN,
+
+    METADATA(ADD_COLUMN, DROP_COLUMN, RENAME_COLUMN, CREATE_TABLE, RENAME_COLUMN),
+    CONNECTOR(SCAN, DELETE, INSERT, METADATA),
```

Comment:
What does CONNECTOR tested feature represent? What does it _not_ represent?
why it does not contain `VIEW`?